Kitodo-DataManagement/src/main/java/org/kitodo/data/database/beans/Process.java

```diff
@@ -399,27 +399,15 @@ public boolean getContainsUnreachableSteps() {
     }
 
     public int getTasksSize() {
-        if (this.getTasks() == null) {
-            return 0;
-        } else {
-            return this.getTasks().size();
-        }
+        return this.getTasks().size();
     }
 
     public int getHistorySize() {
-        if (this.getHistory() == null) {
-            return 0;
-        } else {
-            return this.getHistory().size();
-        }
+        return this.getHistory().size();
     }
 
     public int getPropertiesSize() {
-        if (this.getProperties() == null) {
-            return 0;
-        } else {
-            return this.getProperties().size();
-        }
+        return this.getProperties().size();
     }
 
     public int getWorkpiecesSize() {
```

Comment:
Why did you leave changes for this method out? This method looks like the other changed methods.